config/environments/development.rb

```diff
@@ -70,7 +70,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
-
+  config.hosts << /[a-z0-9\-.]+\.ngrok\.io/
```

Comment:
That's one way to do it, the other one, which can be set at the env level is the CORS setup.